src/core/Object3D.js

```diff
@@ -620,6 +621,11 @@ Object.assign( Object3D.prototype, EventDispatcher.prototype, {
 
 		object.matrix = this.matrix.toArray();
 
+		// SkinnedMesh specific
+
+		if ( this.bindMode !== undefined ) object.bindMode = this.bindMode;
+		if ( this.bindMatrix !== undefined ) object.bindMatrix = this.bindMatrix.toArray();
+
```

Comment:
I just prefer to serialize all public attributes. But your point of view is also valid. The current code is definitely correct. Because i don't have a strong opinion on this, i leave it up to you 😉